samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -69,7 +69,7 @@ public SemanticKernelController(IServiceProvider serviceProvider, IConfiguration
             kernel.RegisterSemanticSkills(semanticSkillsDirectory, this._logger);
         }
 
-        kernel.RegisterNativeSkills(chatRepository, chatMessageRepository, this._promptSettings, this._logger);
+        kernel.RegisterNativeSkills(chatRepository, chatMessageRepository, this._promptSettings, this._configuration, this._logger);
```

Comment:
Use IOption like in Adrian's PR here

Reply:
I will do that if that PR is merged before this one.